SDK/MobileCenter/Microsoft.Azure.Mobile.UWP/IMobileCenterServicePart.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Windows.ApplicationModel.Activation;
+
+namespace Microsoft.Azure.Mobile
+{
+    public partial interface IMobileCenterService
+    {
+        /// <summary>
```

Comment:
This is a callback here, not a user sdk API.
It needs to be renamed OnLaunched in that interface.